neverbleed.c

```diff
@@ -1229,13 +1229,21 @@ __attribute__((noreturn)) static void *daemon_close_notify_thread(void *_close_n
     _exit(0);
 }
 
+#ifdef NEVERBLEED_OPAQUE_RSA_METHOD
+static int (*rsa_finish)(RSA *rsa);
```

Comment:
If you made the change in `neverbleed_init` also for `!NEVERBLEED_OPAQUE_RSA_METHOD`, this declaration should be moved out of `#ifdef`.